adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -780,18 +781,86 @@ public void testResiliencyTokenReturnExtendedLifetimeOnwithoutRetry() throws Pac
             cacheStore.removeAll();
         }
     }
-    
+
+    @SmallTest
+    public void testVerifyManifestPermissionValid() throws InterruptedException, PackageManager.NameNotFoundException {
+        final FileMockContext mockContext = createMockContext();
+        AuthenticationSettings.INSTANCE.setUseBroker(true);
+        final AuthenticationContext authContext = new AuthenticationContext(mockContext,
+                VALID_AUTHORITY, false);
+        final TestAuthCallback callback = new TestAuthCallback();
+        authContext.acquireToken(Mockito.mock(Activity.class), "resource", "clientid", authContext.getRedirectUriForBroker(),
+                "loginHint", callback);
+
+        assertNull(callback.getCallbackException());
```

Comment:
Just out of curiosity, acquiretoken is async call, you know the test is not reliable to pass if we do it this way, right?